jax/_src/pjit.py

```diff
@@ -181,8 +182,8 @@ def _python_pjit_evict_fn():
 
 def _get_fastpath_data(executable, out_tree, args_flat, out_flat, attrs_tracked,
                        ) -> Optional[pxla.MeshExecutableFastpathData]:
+  out_flat_orig = out_flat
   out_flat, out_tree = pxla.reflatten_outputs_for_dispatch(out_tree, out_flat)
```

Comment:
pass orig_out_flag as an argument here?

Reply:
Better yet, take orig_out_flat as the input and pass that in so you don't need to alias.

Reply:
Good call - I renamed the output of the function instead.